skpro/regression/multiquantile.py

```diff
@@ -0,0 +1,362 @@
+"""Multiple quantile regression"""
+
+__all__ = ["MultipleQuantileRegressor"]
+__author__ = ["Ram0nB"]
+
+import numpy as np
+import pandas as pd
+from joblib import Parallel, delayed
+from sklearn.base import clone
+
+from skpro.regression.base import BaseProbaRegressor
+
+
+class MultipleQuantileRegressor(BaseProbaRegressor):
+    """Multiple quantile regressor.
+
+    For quantile regression, often more than one quantile probability is of interest.
+    Therefore, this multiple quantile regressor wraps multiple quantile regressors for
+    multiple quantile probabilities. After fitting, all quantile regressors can be used
+    to construct a quantile/interval prediction for multiple quantile probabilities.
+
+    Parameters
+    ----------
+    mean_regressor : Sklearn compatible regressor, optional
+        Tabular mean regressor for predict.
+    quantile_regressor : Sklearn compatible quantile regressor, optional
+        Tabular quantile regressor. In fit, for every alpha a clone of
+        quantile_regressor is made whereafte the alpha parameter
+        quantile_regressor_alpha_param is set using the set_params method.
+        Subsequently, all regressors are fitted.
+    quantile_regressor_alpha_param : str, optional
```

Comment:
docstring should also list defaults, i.e., what happens if parameter is not passed.